pyface/ui/wx/grid/grid.py

```diff
@@ -103,7 +105,6 @@ class Grid(LayoutWidget):
     default_cell_bg_color = Color("white")
 
     # The default background color to use for read-only cells
-    # default_cell_read_only_color = Trait(Color("linen"), None, Color)
     default_cell_read_only_color = Color(wx.Colour(248, 247, 241))
```

Comment:
Should this one be structured like the other changes in this PR?
```suggestion
    default_cell_read_only_color = Union(None, Color,
                                         default_value=wx.Colour(248, 247, 241))
```

Reply:
I thought about that, but since I removed a commented line and the non-commented code is working, I didn't want to change it and introduce a change in behaviour.

The change here would allow `None` as a value; and I can see that `None` probably isn't good for a trait which is supposed to be holding a default color value.  So I _think_ we're good without further changes.